tests/auto/test_sequential_model.py

```diff
@@ -185,7 +185,59 @@ def test_merge_concat(self):
         nloss = model.evaluate([X_train, X_train], y_train, verbose=0)
         print(nloss)
         assert(loss == nloss)
+    def test_merge_dot(self):
+        print('Test merge: dot')
+        left = Sequential()
+        left.add(Dense(input_dim=input_dim, output_dim=nb_hidden))
+        left.add(Activation('relu'))
+
+        right = Sequential()
+        left.add(Dense(input_dim=input_dim, output_dim=nb_hidden))
+        right.add(Activation('relu'))
+
+        model = Sequential()
+        model.add(Merge([left, right], mode='dot', dot_axes=1))
```

Comment:
Here you are only testing one parameter type of `dot_axes`. Since several types are accepted, they all need to be tested.